CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `MntFlags` and `unmount` on all of the BSDs.
 - Added `any()` and `all()` to `poll::PollFd`.
   ([#1877](https://github.com/nix-rust/nix/pull/1877))
+- Add `PF_ROUTE` to `SockType` on macOS, all of the BSDs.
```

Comment:
```suggestion
- Add `PF_ROUTE` to `SockType` on macOS and the BSDs.
```

Reply:
But isn't this still inaccurate?  Because you also added it for iOS, illumos, Haiku, and Fuchsia, I believe.

Reply:
You are correct, I also moved the entry up the changelog, since these changes have not been released anywhere yet.

Reply:
I think you forgot to push?